virttest/qemu_vm.py

```diff
@@ -1055,6 +1055,16 @@ def add_memorys(devices, params):
                 backend_options["size_mem"] = "%sM" % params["mem"]
                 if params.get("vm_mem_backend"):
                     backend_options["backend_mem"] = params.get("vm_mem_backend")
+                if params.get("vm_mem_backend") == "memory-backend-file":
+                    if params.get("vm_mem_backend_path") == "":
+                        raise virt_vm.VMDeviceError("vm_mem_backend_path is "
+                                                    "empty")
+                    elif not params.get("vm_mem_backend_path"):
+                        raise virt_vm.VMDeviceError("vm_mem_backend_path is "
+                                                    "missing")
```

Comment:
Is it really worth treating the two cases differently?

Reply:
that's follow the discussion on our meeting before, let's wait for others opinion
@yanan-fu @YongxueHong

Reply:
As my understanding, for our discussion before, it is about we can not use `dict[]` method to indicate if the `vm_mem_backend` is available, because the failed scenario `vm_mem_backend =''` will be missed.
For the error situation, i have the same idea with @yihyu.  It is not necessary to distinguish the two failed condition.
And, i would like to suggest using the `VMConfigMissingError`:

```
if not params.get("vm_mem_backend_path"):
    raise virt_vm.VMConfigMissingError(name, 'vm_mem_backend_path')
```